Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutVaultRequest.swift

```diff
@@ -0,0 +1,59 @@
+#if canImport(BraintreePayPal)
+import BraintreePayPal
+#endif
+
+/**
+ Options for the PayPal Vault flow.
+ */
+@objc public class BTPayPalNativeVaultRequest: BTPayPalVaultRequest, BTPayPalNativeRequest {
+
+    let hermesPath: String
+
+    let paymentType: BTPayPalPaymentType = .vault
+
+    /**
+     Initializes a PayPal Vault request.
+
+     - Returns: A PayPal Vault request.
+     */
+    @objc public override init() {
+        self.hermesPath = "v1/paypal_hermes/setup_billing_agreement"
```

Comment:
nit: Is it possible to combine the property declaration and assignment for `hermesPath` and remove the `init` override?

Reply:
Yep, good call. updated!